### DISCUSSION TITLE
Cue sheet plugin bug

### DISCUSSION THREAD

**Discussion by dprimon** [audacious-media-player/audacious#69] [General]
Hi,
   I recently tried to add a cue sheet to a playlist but only obtained a single entry . Looking at the cue sheet with a text editor I eventually realized that the file name of the audio image contained parenthesis "()" so I adjusted the audio image file name and cue sheet to a filename which did not include parenthesis. Probably the plugin should be adjusted to handle filenames with such characters.

P.S.: I've been advised to write here since redmine is having issues with my email address hence I couldn't properly file a bug. Using Audacious 4.3.1 from Archlinux.

**Comment by jlindgren90** [comment]
I tried to reproduce but don't see the issue. This sample worked fine for me:
```
PERFORMER "Performer"
TITLE "Title"
REM DATE 2014
FILE "test().mp3" WAVE
  TRACK 01 AUDIO
    TITLE "Song 01 Track 01"
    INDEX 01 00:00:00
  TRACK 02 AUDIO
    TITLE "Song 01 Track 02"
    INDEX 01 00:30:00
  TRACK 03 AUDIO
    TITLE "Song 01 Track 03"
    INDEX 01 01:00:00
```

**Reply by dprimon** [reply]
Oh... I've gone through the original filenames before I renamed them and realized the archived file had a typo... I just missed that... Sorry for bothering
